.gitignore

```diff
@@ -69,7 +69,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+.env*
```

Comment:
Do we still need `.env.test` if you use the asterisk sign?

Reply:
Yes, I just try not to push `.env` file by accident. I'll modify it back to the original.